src/Platform/Microsoft.Testing.Platform/Hosts/CommonTestHost.cs

```diff
@@ -56,12 +53,23 @@ public async Task<int> RunAsync()
             await DisposeServiceProviderAsync(ServiceProvider, isProcessShutdown: true);
             await DisposeHelper.DisposeAsync(ServiceProvider.GetService<FileLoggerProvider>());
             await DisposeHelper.DisposeAsync(ServiceProvider.GetTestApplicationCancellationTokenSource());
-            await DisposeHelper.DisposeAsync(dotnetTestPipeClient);
+            await DisposeHelper.DisposeAsync(DotnetTestConnection);
         }
 
         return exitCode;
     }
 
+    private string GetHostType()
+    {
+        string hostType = this switch
```

Comment:
pls add a comment:

For now we don't inherit `TestHostOrchestratorHost` from the common one so we don't connect when we orchestrate.